src/dapps/DappsScreen.tsx

```diff
@@ -240,8 +240,6 @@ function DappsScreen({ navigation }: Props) {
                 <FilterChipsCarousel
                   chips={filterChips}
                   onSelectChip={handleToggleFilterChip}
-                  primaryColor={Colors.successDark}
```

Comment:
This defaults to gray/black buttons, Kayla mentioned wanting this standard across the app [here](https://valora-app.slack.com/archives/C03D3K379FE/p1721399533859949?thread_ts=1721341291.445169&cid=C03D3K379FE)